docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -215,54 +213,117 @@ for more information about the environment variables.
 
 The index name to write events to. The default is
 +"{beatname_lc}-%\{[beat.version]\}-%\{+yyyy.MM.dd\}"+ (for example,
-+"{beatname_lc}-{version}-2017.04.26"+). If you change this setting, you also
++"{beatname_lc}-{version}-{localdate}"+). If you change this setting, you also
 need to configure the `setup.template.name` and `setup.template.pattern` options
 (see <<configuration-template>>).
+ifndef::no_dashboards[]
+If you are using the pre-built Kibana
+dashboards, you also need to set the `setup.dashboards.index` option (see
+<<configuration-dashboards>>).
+endif::no_dashboards[]
 
-When loading the pre-built Kibana dashboards via {beatname_uc},
-you also need to set the `setup.dashboards.index` option (see <<configuration-dashboards>>).
-
+You can set the index dynamically by using a format string to access any event
+field. For example, this configuration uses a custom field, `fields.log_type`,
+to set the index: 
 
-===== `indices`
+["source","yaml",subs="attributes"]
+------------------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["http://localhost:9200"]
+  index: "%\{[fields.log_type]\}-%\{[beat.version]\}-%\{+yyyy.MM.dd}\" <1>
+------------------------------------------------------------------------------
 
-Array of index selector rules supporting conditionals, format string
-based field access and name mappings. The first rule matching will be used to
-set the `index` for the event to be published. If `indices` is missing or no
-rule matches, the `index` field will be used.
+<1> We recommend including `beat.version` in the name to avoid mapping issues
+when you upgrade.
 
-Rule settings:
+With this configuration, all events with `log_type: normal` are sent to an 
+index named +normal-{version}-{localdate}+, and all events with
+`log_type: critical` are sent to an index named
++critical-{version}-{localdate}+.
 
-*`index`*: The index format string to use. If the fields used are missing, the rule fails.
+TIP: To learn how to add custom fields to events, see the
+<<libbeat-configuration-fields,`fields`>> option.  
 
-*`mapping`*: Dictionary mapping index names to new names
+See the <<indices-option-es,`indices`>> setting for other ways to set the index
+dynamically.
 
-*`default`*: Default string value if `mapping` does not find a match.
 
-*`when`*: Condition which must succeed in order to execute the current rule.
+ifdef::deprecate_dashboard_loading[]
 
-ifeval::["{beatname_lc}" == "apm-server"]
-NOTE: When changing APM indices, you have to update the {kibana-ref}/apm-settings-kb.html[Kibana APM settings]
-accordingly to ensure the Kibana UI can still query the data.
+deprecated[{deprecate_dashboard_loading}]
 
 endif::[]
 
-Examples elasticsearch output with `indices`:
+[[indices-option-es]]
```

Comment:
we don't support these selector rules, so this entire section should be removed for us

Reply:
L257-L323

Reply:
Removed in ccae958

Reply:
ok, i actually thought we had agreed previously to not include it.